.github/workflows/ci.yml

```diff
@@ -34,32 +34,53 @@ jobs:
         name: Type Check
         runs-on: ubuntu-latest
         steps:
-            - uses: actions/checkout@v2
+            - uses: actions/checkout@v3
             - name: Setup Node
-              uses: actions/setup-node@v1
+              uses: actions/setup-node@v3
               with:
                   node-version: ${{env.NODE_VERSION}}
             - run: npm ci
             - run: npm run typeCheck --if-present
-    build:
-        if: github.ref == 'refs/heads/main' || startsWith(github.ref, 'refs/tags/v')
+
+    publish-dev:
+        if: github.ref == 'refs/heads/main'
         needs: [lint, typeCheck, test]
         runs-on: ubuntu-latest
         steps:
-            - uses: actions/checkout@v2
-            - uses: actions/setup-node@v1
+            - uses: actions/checkout@v3
+            - uses: actions/setup-node@v3
               with:
                   node-version: ${{env.NODE_VERSION}}
             - run: npm ci
             - run: npm run build
+            - run: mkdir dev-build
+            - run: |
+                cp -r es dev-build | 
+                cp -r lib dev-build | 
+                cp -r dist dev-build |
+                cp -r style dev-build |
+                cp -r type-declarations dev-build |
+                cp README.md dev-build |
+                cp package.json dev-build
```

Comment:
all these cp lines look a bit clunky - it would be cool if they were encapsulated in a `npm run copy_build_artifacts` script so that this file doesn't have to know about all those directories.

Reply:
I'll try using this: https://www.npmjs.com/package/copyfiles

Reply:
ok I got it to work. there are a lot of commits because I mistakenly thought the dist and lib folders needed to be in there too and I couldn't figure out why they were getting copied